src/plugins/mer/meremulatoroptionswidget.cpp

```diff
@@ -376,6 +376,13 @@ void MerEmulatorOptionsWidget::onAddButtonClicked()
     };
     Sdk::createEmulator(dialog.selectedVmUri(), &loop, whenDone);
     loop.exec();
+    if (!emulator) {
+        QMessageBox::warning(this,
+                tr("Could not add a %1 Emulator").arg(Sdk::osVariant()),
+                tr("The selected virtual machine could not be configured to be used as a %1 Emulator")
+                    .arg(Sdk::osVariant()));
+        return;
+    }
     QTC_ASSERT(emulator, return);
```

Comment:
I forgot to mention this can be removed now.

Reply:
How on earth did I not notice that myself...